app/models/cohort.rb

```diff
@@ -12,6 +12,10 @@ def self.current
     find_by(start_year: 2021)
   end
 
+  def self.next
+    find_by(start_year: 2022)
+  end
```

Comment:
Certainly not a deal-breaker, but have you considered this? You might be able to omit most of your feature conditions elsewhere.

```suggestion
  def self.next
    find_by(start_year: FeatureFlag.active?(:multiple_cohorts) ? 2022 : 2021)
  end
```

Reply:
Good suggestion. I pushed the changes.